pq-crypto/sike_r3/sikep434r3_fp_x64_asm.S

```diff
@@ -0,0 +1,1050 @@
+//*******************************************************************************************
```

Comment:
Exclude this entire file from the pull request since you're not adding the assembly implementation right now.